packages/circuit-ui/components/Avatar/Avatar.docs.mdx

```diff
@@ -0,0 +1,46 @@
+import { Status, Props, Story } from '../../../../.storybook/components';
+
+# Avatar
+
+<Status.Stable />
+
+The Avatar component displays an identity or an object image. It can be passed to the [ImageInput](Forms/ImageInput) to allow users to upload an avatar.
+
+<Story id="components-avatar--base" />
+
+<Props />
+
+## Usage guidelines
+
+- **Do** use the right variant for your use case (see _Variants_ below).
+- **Do** use the [ImageInput component](Forms/ImageInput) with the `component={Avatar}` prop to allow users to upload an avatar (note: the ImageInput only supports `variant="object"` for now).
+
+## Accessibility
```

Comment:
Thanks for adding a dedicated _Accessibility_ section! 🙌🏻

Some feedback: I would flip the order of the recommendations: adding alt text should be the default, omitting it the exception. It's better to have duplicate content than missing content.

Reply:
I reworded it to put more focus on when the alt text was necessary.

Do you think we should make `alt` a required prop and remove the `""` default? In that case developers will need to explicitly pass `""` if their images are presentational.

Reply:
I think that's a good idea. It forces developers to make a conscious decision.